### DISCUSSION TITLE
Howto shell script: change user password

### DISCUSSION THREAD

**Discussion by ron-from-nl** [nilsteampassnet/TeamPass#3803] [Q&A]
Hi, not sure if my question is a hard or maybe not so hard one to answer, but here is the situation:

Say TeamPass has two users:

An Admin called: admin (only administator tasks)
A Manager called: steve (creates and maintains all the secret items in the folders)

How can I (from the command-line) bash / php script: [Change login password ] for these two users: admin, steve ?

The server:
TeamPass 3.0.9
Debian GNU/Linux 11 (bullseye)
PHP version: 7.4.33
Server version: 5.5.5-10.5.19-MariaDB-0+deb11u2

The purpose behind this is a single-sign-on user on a web server.
TeamPass is the last webapp (among others) to include in a password change script.
It would be really cool if a shell script could change the TeamPass user passwords.

Is there anyone that knows howto do this ? 

Cheers, Ron

**Comment by nilsteampassnet** [comment]
@ron-from-nl 
I only so this topic now.
Could indeed be nice to have such script.
How do you see this to work?
The change of user password is done without providing the old one?
An authorization key would be required as a parameter in the instruction, correct?
An email is send to user to infor his password has been changed, correct?
Should not be possible for AD user, only local user.

**Comment by Arash9703** [comment]
@nilsteampassnet  Je te remercie pour ton avis ! 
https://github.com/nilsteampassnet/TeamPass/discussions/3792#discussioncomment-7777423
![image](https://github.com/nilsteampassnet/TeamPass/assets/143269985/dae8bff7-526a-456a-8b08-6e4cce32f07a)
